projects/rero/ng-core/src/lib/text-read-more/text-read-more.component.html

```diff
@@ -0,0 +1,11 @@
+<ng-container>
+  <div class="d-inline" [innerHTML]="this.textToDisplay | nl2br"></div>
+  <a class="pl-2" href="#" *ngIf="this.isLimitReached" (click)="toggleText($event)">
+    <ng-container *ngIf="!isExpanded; else expanded">
+      {{ this.showMoreLabel }}
+    </ng-container>
+    <ng-template #expanded>
+      {{ this.showLessLabel }}
+    </ng-template>
```

Comment:
Why you make this on some variables ? `this.textToDisplay`, `this.isLimitReached` and `this.showLessLabel`

Reply:
done